modules/i2c/i2cs.h

```diff
@@ -0,0 +1,50 @@
+/** @defgroup i2c I2C
+ * @brief I2C module
+ */
+//@{
+/**
+ * @file
+ * @brief I2Cs definitions
```

Comment:
That's a lazy change (what is the meaning of *I2Cs*?). *I2C slave definitions* would be better.